.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Storybook Debug",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run storybook",
```

Comment:
I'll discuss with @vojtechszocs if we're comfortable adding editor specific helpers to the SDK. For now I'll merge this in and we can remove it later if necessary.